android/app/src/main/java/com/epfl/dedis/hbt/ui/wallet/ShowQrFragment.kt

```diff
@@ -79,15 +78,16 @@ class ShowQrFragment : Fragment() {
                 is ReceiverShow -> walletViewModel.transitionTo(
                     ReceiverRead(
                         PendingTransaction(
-                            walletViewModel.user!!.passport,
+                            walletViewModel.user.name,
```

Comment:
It might be a good idea to keep the passport number (which is unique for the transaction).